packages/front-end/components/Features/ConditionInput.tsx

```diff
@@ -322,6 +349,30 @@ export default function ConditionInput(props: Props) {
                       className={styles.matchingInput}
                       containerClassName="col-sm-12 col-md mb-2"
                     />
+                  ) : attribute.datatype === "date" ? (
+                    <div>
+                      <Field
+                        type="datetime-local"
+                        value={localDateTime}
+                        onKeyDown={(e) => e.preventDefault()}
```

Comment:
V hacky, but I'm running into a bug where if a user manually edits the year, the datetime-local picker breaks.